content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+there are the usual rough edges of anything new, we believe that it went well.
+To that end, we've just started the Scoping phase for Gateway API 1.3, which
+will end on November 19, 2024, with the release expected around the end of
+January 2025.
+
+### `gwctl` Moves Out
+
+The `gwctl` CLI tool has moved into its very own repository,
+https://github.com/kubernetes-sigs/gwctl. `gwctl` has proven a valuable tool
+for the Gateway API community; moving it into its own repository will, we
+believe, make it easier to maintain and develop. As always, we welcome
+contributions; `gwctl` makes quite a few things about Gateway API and
+Kubernetes easier to manage.
+
+### Maintainer Changes
+
+Rounding out our changes to the project itself, we're pleased to announce that
+[Mattia Lavacca] has joined the ranks of Gateway API Maintainers! We're also
+sad to announce that [Keith Mattix] has stepped down as a GAMMA lead --
+happily, [Mike Morris] has returned to the role. We're grateful for everything
+Keith has done, and excited to have Mattia and Mike on board.
+
+[Mattia Lavacca]: https://github.com/mlavacca
+[Keith Mattix]: https://github.com/@keithmattix
+[Mike Morris]: https://github.com/@mikemorris
+
+## Gateway API background
+
+The idea of Gateway API was initially [proposed](https://youtu.be/Ne9UJL6irXY?si=wgtC9w8PMB5ZHil2)
+at the 2019 KubeCon San Diego as the next generation
+of Ingress API. Since then, an incredible community has formed to develop what
+has likely become the
+[most collaborative API in Kubernetes history](https://www.youtube.com/watch?v=V3Vu_FWb4l4).
+Over 200 people have contributed to this API so far, and that number continues to grow.
+
+The maintainers would like to thank _everyone_ who's contributed to Gateway API, whether in the
+form of commits to the repo, discussion, ideas, or general support. We literally
+couldn't have gotten this far without the support of this dedicated and active
+community.
+
+## Try it out
+
+Unlike other Kubernetes APIs, you don't need to upgrade to the latest version of
+Kubernetes to get the latest version of Gateway API. As long as you're running
+Kubernetes 1.26 or later, you'll be able to get up and running with this
+version of Gateway API.
+
+To try out the API, follow our [Getting Started Guide](https://gateway-api.sigs.k8s.io/guides/).
+
+## Get involved
+
+There are lots of opportunities to get involved and help define the future of
+Kubernetes routing APIs for both ingress and service mesh.
+
+* Check out the [user guides](https://gateway-api.sigs.k8s.io/guides) to see what use-cases can be addressed.
+* Try out one of the [existing Gateway controllers](https://gateway-api.sigs.k8s.io/implementations/).
+* Or [join us in the community](https://gateway-api.sigs.k8s.io/contributing/)
+  and help us build the future of Gateway API together!
+
+## Related Kubernetes blog articles
+
+* [New Experimental Features in Gateway API v1.0](/blog/2023/11/28/gateway-api-ga/)
```

Comment:
Recommend adding a link to https://kubernetes.io/blog/2024/05/09/gateway-api-v1-1/

Reply:
Ouch, good catch.